### PR TITLE
Adding /free instructions to busy-message

### PR DESCRIPTION
**Pull request by PajserDivizija**
Improving the UX of the automatic busy-message by adding the instructions of how to free the channel again.

### CODE REVIEW

File: application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
             Thank you for asking in an available channel. A helper will be with you shortly!
 
             __Do not post your question in other channels.__
+                 
+            After you are satisfied with the given answer don't forget to free the channel for others with the '/free' command!
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java:16]: Imo this sounds kinda harsh, with the exclamation mark and the "dont foget to".

I would rephrase it and shorten to:

> Once your issue is resolved, feel free to `/free` the channel, thanks.

**Review by Zabuzard — CHANGES_REQUESTED** [review]

**Comment by Zabuzard** [comment]
Spotless doesnt pass, please make sure to run spotless, thanks.

File: application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java (hunk 1 above)
Comment (Tais993) [reply application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java:16]: +1

File: application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
             __Do not post your question in other channels.__
                  
-            After you are satisfied with the given answer don't forget to free the channel for others with the '/free' command!
+            Once your issue is resolved, feel free to /free the channel, thanks.
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java:16]: I think we should put the `/free` into code-highlighting, like \`/free\`. Otherwise, I like it, thanks.

**Review by Zabuzard — CHANGES_REQUESTED** [review]

**Review by Heatmanofurioso — APPROVED** [review]
Apart from Zabu's comment
Looks fine!

**Comment by Zabuzard** [comment]
(Spotless is still not passing, please execute spotless, thanks)

**Comment by PajserDivizija** [comment]
> (Spotless is still not passing, please execute spotless, thanks)

I'm not quite sure how to fix this. I tried to search for a solution but didn't find anything. Looking at the output log it suggests I run a command but since I don't have a terminal to spotless I don't know how to execute the command

**Comment by Zabuzard** [comment]
> > (Spotless is still not passing, please execute spotless, thanks)
> 
> I'm not quite sure how to fix this. I tried to search for a solution but didn't find anything. Looking at the output log it suggests I run a command but since I don't have a terminal to spotless I don't know how to execute the command

You can have a look here in our Wiki: [Code-Guidelines#run-spotless](https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines#run-spotless)

But essentially it is just about executing the gradle goal. So `gradle: spotlessApply` (or clicking the buttons in the UI of ur IDE).

**Comment by PajserDivizija** [comment]
> You can have a look here in our Wiki: [Code-Guidelines#run-spotless](https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines#run-spotless)
> 
> But essentially it is just about executing the gradle goal. So `gradle: spotlessApply` (or clicking the buttons in the UI of ur IDE).

Is it possible to run this from within GitHub? (sorry for maybe a dumb question but I don't have a very detailed knowledge of the quirks and features of GitHub). I edited this code directly from my user agent but your suggestion would require pulling, installing dependencies and solving import errors which is a lot more time consuming in the time of exams on university

**Comment by java-coding-prodigy** [comment]
> Is it possible to run this from within GitHub? (sorry for maybe a dumb question but I don't have a very detailed knowledge of the quirks and features of GitHub). I edited this code directly from my user agent but your suggestion would require pulling, installing dependencies and solving import errors which is a lot more time consuming in the time of exams on university

There are multiple ways to run that command, you can just execute the gradle task, or if you are having trouble locating that, you can open your terminal(cmd for example), navigate to the projects directory, and run `gradlew spotlessApply`

**Comment by Zabuzard** [comment]
> > You can have a look here in our Wiki: [Code-Guidelines#run-spotless](https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines#run-spotless)
> > But essentially it is just about executing the gradle goal. So `gradle: spotlessApply` (or clicking the buttons in the UI of ur IDE).
> 
> Is it possible to run this from within GitHub? (sorry for maybe a dumb question but I don't have a very detailed knowledge of the quirks and features of GitHub). I edited this code directly from my user agent but your suggestion would require pulling, installing dependencies and solving import errors which is a lot more time consuming in the time of exams on university

Oh, interesting. This indeed requires you to have the full thing properly setup on your machine - I wasnt aware you made this edit from a pure text editor without a proper setup.

I will do the change and propose a PR to your fork, which you can accept and then it should leak over here as well.

**Comment by Zabuzard** [comment]
I was not able to do the change to your PR directly, it was necessary to create a new branch that I can edit myself. Which means I had to open a new PR based on your branch.

Hence, closing this in favor of the "cloned" #353 .
